src/dotnet/Program.cs

```diff
@@ -129,13 +139,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 exitCode = result.ExitCode;
             }
 
-            telemetryClient.TrackEvent(
-                command,
-                null,
-                new Dictionary<string, double>
-                {
-                    ["ExitCode"] = exitCode
```

Comment:
I asked this in a separate comment, but I think it is worth repeating here:

"What are we going to use this telemetry information for?"

I don't think anyone has answered that question, and that's why we are having questions like this pop up in the implementation.

Is our goal to use telemetry to understand if customers are successful using our product?